configure.py

```diff
@@ -774,6 +774,7 @@ def find_ninja():
                         help='List all available build artifacts, that can be passed to --with')
 arg_parser.add_argument('--date-stamp', dest='date_stamp', type=str,
                         help='Set datestamp for SCYLLA-VERSION-GEN')
+arg_parser.add_argument('--with-cmake', action='store_true', help='Create building system with CMake')
```

Comment:
I think a better name for this option would be `--use-cmake`. `--with-*` is usually used for including artifacts in the build.

I think the message should be: "Use CMake as the build system".

Reply:
sure. will do.

Reply:
a little bit background: as both ninja and CMake are claimed to be build systems. while, strictly speaking, CMake generates a build system. but yeah, and it would be even more confusing if one tries to differentiate one from another.